test/elements.test.ts

```diff
@@ -0,0 +1,37 @@
+import { ElementDictionary } from "../src/data/elements";
+import { Phases, Phase } from "../src/interfaces/element-props";
+
+const elements: string[] = Object.keys(ElementDictionary);
+
+function checkUrl(url: string): boolean {
+  try {
+    new URL(url);
+  } catch (e) {
+    return false;
+  }
+  return true;
+}
+
+function checkPhase(phase: Phase): boolean {
+  return Object.values(Phases).includes(phase);
+}
+
+describe("Periodic Table Elements", () => {
+  it("should return true for correct URL of source", () => {
+    for (let el of elements) {
+      expect(checkUrl(ElementDictionary[el].source)).toBe(true);
+    }
+  });
+  it("should return true for correct URL of Spectral Image", () => {
+    for (let el of elements) {
+      if (typeof ElementDictionary[el].spectralImage === "string") {
```

Comment:
Can falsy test cases be written too where `url` is not present? Are there elements where url might be absent?

Reply:
yes , for this property some urls are missing .

Reply:
I think you can write test cases or them too

Reply:
Test case looks fine

Reply:
@maneetgoyal sir so, we dont have to write test cases for missing values ?

Reply:
Yeah the current form is okay